### PR TITLE
3/rich links

### PR DESCRIPTION
**Pull request by stuartromanek**
B5EQgBnYBG

This experience

https://apostrophecms.atlassian.net/browse/A30U-551?atlOrigin=eyJpIjoiZmI5ODM3NmUwZjA5NDIyMDkyNTlmODhjYmFmNzM4M2EiLCJwIjoiaiJ9
https://apostrophecms.atlassian.net/browse/A30U-550?atlOrigin=eyJpIjoiMDI1NDYzMGFmMjgxNGM3ZjhkNzM4ZGIzMjg1YmVkMjIiLCJwIjoiaiJ9
https://apostrophecms.atlassian.net/browse/A30U-564?atlOrigin=eyJpIjoiN2NjODI4YWM5NDViNGNjMDk4NDIxNTQ2MmE4ZGQ2MjciLCJwIjoiaiJ9
https://apostrophecms.atlassian.net/browse/A30U-562?atlOrigin=eyJpIjoiZTgyOGU4OGYwN2Q0NDQ2YmJkMzVmYjgwZmNhZjc2NDQiLCJwIjoiaiJ9
https://apostrophecms.atlassian.net/browse/A30U-563?atlOrigin=eyJpIjoiNTEyZWQzMTMwNjJlNGQyNTg0MDA0MjlkNzNjZjJmNzIiLCJwIjoiaiJ9

### CODE REVIEW

File: modules/@apostrophecms/rich-text-widget/ui/apos/components/ApostropheTiptapLink.vue [hunk 1]
```diff
@@ -1,89 +1,218 @@
...
+        },
+        {
+          name: 'target',
+          label: 'Target',
+          help: 'Where should this link open?',
+          type: 'select',
+          def: '_self',
+          choices: [
+            {
+              label: 'Current browsing context (_self)',
```
Comment (localghost443) [inline modules/@apostrophecms/rich-text-widget/ui/apos/components/ApostropheTiptapLink.vue:114]: Minor, but can we just simplify this a bit? Can we just have "Current Tab" and "New Tab"? Its a bit technical as written

**Review by localghost443 — CHANGES_REQUESTED** [review]

File: modules/@apostrophecms/rich-text-widget/ui/apos/components/ApostropheTiptapLink.vue (hunk 1 above)
Comment (stuartromanek) [reply modules/@apostrophecms/rich-text-widget/ui/apos/components/ApostropheTiptapLink.vue:114]: Yeah open to better labels on these, i think i either c+p'd mdn or paraphrased

Comment (stuartromanek) [reply modules/@apostrophecms/rich-text-widget/ui/apos/components/ApostropheTiptapLink.vue]: updated this

**Review by abea — APPROVED** [review]

**Review by localghost443 — APPROVED** [review]

File: modules/@apostrophecms/rich-text-widget/ui/apos/components/ApostropheRichTextWidgetEditor.vue [hunk 2]
```diff
@@ -2,8 +2,9 @@
   <div class="apos-richtext-editor">
     <editor-menu-bar :editor="editor">
       <div class="apos-richtext-menubar" slot-scope="{ commands, isActive }">
-        <component v-for="item in toolbar"
-          v-bind:key="item"
+        <component
+          v-for="(item, index) in toolbar"
+          :key="item + '-' + index"
```
[the remark this answers is not in the document]
Comment (stuartromanek) [reply modules/@apostrophecms/rich-text-widget/ui/apos/components/ApostropheRichTextWidgetEditor.vue:7]: yeah, each object is a component, so Divider gets repeated a bunch and cries about dup keys

**Pull request merged**